online-dashboard-api/app/Eums/CustomerEventStatus.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Enums;
+
+/**
+ * @model App\Models\CustomerEventLog
+ *
+ * @column status
+ */
+final class CustomerEventStatus extends BaseEnum
```

Comment:
Rename the file to generic so that we can use this enums for other purposes too!